src/promise.ts

```diff
@@ -119,3 +130,22 @@ Promise.sequence = function sequence<A>(as: Array<Promise<A>>): Promise<Array<A>
     });
   });
 };
+
+Promise.all = Promise.sequence
```

Comment:
IS IT THOUGH?!

Reply:
Let's see... It... Creates an unset array, walks over all the promises... Writes in place in the results-array as they're done... Returns when all are resolved... rejects immediately on Error... 

YES IT FUCKING WELL IS